package.json

```diff
@@ -19,6 +19,8 @@
     "flow:stop": "flow stop",
     "eslint": "eslint .",
     "eslint:fix": "eslint . --fix",
+    "deploy:now": "now --dotenv",
+    "deploy:updateURL": "yarn babel-node scripts/deploymentUtils.js updateLiveURL",
```

Comment:
Shouldn't these scripts be probably added to Circle CI config?

Reply:
I agree that's certainly a good option. I left it non-automated for now, so we can manage deployments manually. If automatic re-deployment is desired on CircleCI right now. We would only need to decide whose `now.sh` account we use to do the deployment?

Reply:
Then I'm missing the point of this PR little bit, especially `deploy:updateURL`  🙂

Reply:
ok, I see the description, I can merge it, but it's still not optimal unless it's part of CI

Reply:
Ok! I will use my own now.sh token to deploy via circleCI